solr/Dockerfile

```diff
@@ -3,11 +3,13 @@ ARG repository=local
 ARG tag=latest
 FROM --platform=$BUILDPLATFORM ${repository}/download:${tag} AS download
 
+ENV SOLR_VERSION="8.11.2"
+ENV SOLR_KEYS="86EDB9C33B8517228E88A8F93E48C0C6EF362B9E"
```

Comment:
`SOLR_KEYS` is not used anywhere here?

Reply:
No, this is an attempt to make it easier/consistent for developers to update without needing to modify strings embedded within logic. By consistent I'm assuming this is the right way to approach this and modify other Dockerfiles in the future to match.

Reply:
If it's not being used, let's just remove it. As it's an extra step that has no effect.

Reply:
The documentation in the README.md should also get updated to not reference `SOLR_KEYS` after it's removed.

Reply:
OK, done